package.json

```diff
@@ -69,28 +71,28 @@
     "eslint-loader": "^4.0.2",
     "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-jsx-a11y": "^6.10.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react": "^7.37.2",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-webpack-plugin": "^4.2.0",
     "flatpickr": "^4.6.13",
     "json": "^11.0.0",
     "markdownlint-cli2": "^0.4.0",
     "postcss": "^8.4.47",
-    "prettier": "^2.7.1",
+    "prettier": "^3.3.3",
```

Comment:
We need to upgrade Prettier to make [storybook-addon-html](https://github.com/whitespace-se/storybook-addon-html?tab=readme-ov-file#requirements) work with Storybook 8, which is why Stylelint also has been upgraded.